tests/inspection_tests.py

```diff
@@ -186,6 +206,8 @@ def get_service_health(service_host):
                 test_passed = False 
             else:
                 should_fail_tests.append(inspection_test["title"])
+                if "invalid_blob_token" in inspection_test or "no_container_blob" in inspection_test or "bad_blob_endpoint" in inspection_test:
```

Comment:
This is a bit awkward as it will need to keep adding each additional test.  Perhaps instead we should keep the original variable unchanged (e.g. `blob_upload_sas_url`) and then use a modified copy of it in each test above?  for example:

```
       blob_sas_to_use = blob_upload_sas_url
       if "invalid_blob_token" in inspection_test:
             blob_sas_to_use =  blob_sas_to_use.replace(urllib.parse.urlparse(blob_upload_sas_url).query, "")

...

        elif "blob_upload" in inspection_test and inspection_test["blob_upload"] == True:
            DATA = urllib.parse.urlencode({"saskey":storage_sas, "blobsasurl":blob_sas_to_use})

```
then we don't need this reset `if` statement at all...

Reply:
Good suggestion. Updated.